### PR TITLE
build: Provide setup script for running as a systemd service

### PR DESCRIPTION
**Pull request by travipross**
I've created a script which I think will walk the user through the process of installing as a service. I've tested a few times in several variations on my machine and it seems to work great, so I think it's good to include.

### CODE REVIEW

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:36]: Will circle back to this before merging

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:22]: This is what I came up with to help encourage usage within a virtual environment, while not explicitly preventing someone from ignoring that advice. 

I use `pyenv` and it should work equally well with `venv`. Please do test if you'd like though.

File: resources/poglink.service
Comment (travipross) [inline resources/poglink.service:12]: Just using the `##` as something distinct to look for when doing `sed` substitutions in the install script.

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:11]: If `FORCE=true` then this block will pass and it will install `poglink` on the system Python environment. This might be desireable if using a dedicated VM for this bot and didn't want to bother with a `venv` when you knew there was nothing to worry about conflicting with.

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:18]: Didn't want to pin a specific version or force an update; I feel like that should be up to the user to decide if they want a specific version or want to update.

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:41]: I like to use `install` instead of `cp` here because it allows me to set a backup option, in case you run accidentally and overwrite a service file you've modified.

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:46]: Just a simple search and replace for the delimited tokens I've defined.

File: scripts/install-service.sh
Comment (travipross) [inline scripts/install-service.sh:55]: This will leave existing config files alone if they are already found in the data directory.

**Comment by travipross** [comment]
Housekeeping reminder @FM-17

**Review by FM-17 — APPROVED** [review]
Finally got around to reviewing this. It looks great. I'd say we're good to merge pending [this](https://github.com/FM-17/poglink/pull/35#discussion_r768341234) follow-up, and any other changes that may be needed to bring this up to date.

**Comment by FM-17** [comment]
@travipross housekeeping reminder

**Comment by travipross** [comment]
> > Housekeeping reminder @FM-17
> 
> Housekeeping reminder @travipross

Elaboarating on the housekeeping reminder: Want to circle back to [this](https://github.com/FM-17/poglink/pull/35#discussion_r768341234) before merging.

**Comment by travipross** [comment]
> Elaboarating on the housekeeping reminder: Want to circle back to [this](https://github.com/FM-17/poglink/pull/35#discussion_r768341234) before merging.

Actually, nevermind. I had mostly already addressed that concern and I feel the concept will be subject to further review anyway when looking at #41. Merging now.

**Pull request merged**
